packages/server-wallet/src/models/channel.ts

```diff
@@ -406,17 +406,21 @@ export class Channel extends Model implements ChannelColumns {
   }
 
   /**
-   * Is a prefund state (or later) supported
+   * Have all participants signed a preFund state?
```

Comment:
It's not exactly this - e.g. we don't want them to have signed conflicting preFunds. Maybe this should just read "is the preFund setup stage complete".

Reply:
Good point. I have inserted the word "consistent" in cd14064 to try and justice to that. 

I think it's nice to have the docstring say more than is already said by the name of the function.